gcovr/formats/html/write.py

```diff
@@ -69,16 +69,23 @@ def reuse_value(*args):
     def __call__(self, *args):
         return self.get(*args)
 
+# html_theme string is <theme_directory>.<color> or only <color> (if only color the use default)
+# examples: github.green github.blue or blue or green
+def get_theme_name(html_theme):
+    return html_theme.split(".")[0]  if "." in html_theme else "default"
+
+def get_theme_color(html_theme):
+    return html_theme.split(".")[1]  if "." in html_theme else html_theme
 
 # Loading Jinja and preparing the environmen is fairly costly.
 # Only do this work if templates are actually used.
 # This speeds up text and XML output.
 @Lazy
-def templates():
+def templates(options):
     from jinja2 import Environment, PackageLoader
 
     return Environment(
-        loader=PackageLoader("gcovr.formats.html"),
+        loader=PackageLoader("gcovr.formats.html.%s" % get_theme_name(options.html_theme)),
```

Comment:
Please use f-strings.
```suggestion
        loader=PackageLoader(f"gcovr.formats.html.{get_theme_name(options.html_theme)}"),
```